stdlib/bytes.ml

```diff
@@ -15,20 +15,20 @@
 
 (* Byte sequence operations *)
 
-external length : bytes -> int = "%string_length"
+external length : bytes -> int = "%bytes_length"
 external string_length : string -> int = "%string_length"
-external get : bytes -> int -> char = "%string_safe_get"
-external set : bytes -> int -> char -> unit = "%string_safe_set"
+external get : bytes -> int -> char = "%bytes_safe_get"
+external set : bytes -> int -> char -> unit = "%bytes_safe_set"
 external create : int -> bytes = "caml_create_string"
```

Comment:
you've started partially duplicating string primitives : `caml_ml_output_bytes` `caml_blit_bytes` `caml_caml_bytes_equal`.
I would duplicate all or none.